tests/js/py2js/object-methods.simple.failing

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file        py2js/object.simple
+ *              Simple test which shows that sending objects from Python to JS still retains
+ *              the ability to perform basic Object methods like Object.keys(), Object.values() and
+ *              Object.entries() on them.
+ * @author      Joash Mathew, <joash@distributive.network>
+ * @date        July 2023
+ */
+'use strict';
+
+const obj = python.eval('{ "a": 1, "b": 2, "c": 3 }');
+const throughJS = (x) => x;
+const jsObj = throughJS(obj);
+const standardJSObj = { a: 1, b: 2, c: 3 };
+
+if (Object.keys(jsObj) !== Object.keys(standardJSObj))
```

Comment:
Array cannot be compared directly in JS.
`==` only matches if the two JS Arrays have the same memory address.

Reply:
```suggestion
if (JSON.stringify(Object.keys(jsObj)) !== JSON.stringify(Object.keys(standardJSObj)))
```